test/test_asymetric.py

```diff
@@ -151,6 +153,25 @@ def test_RSA_OAEP(self):
 
         self.assertEqual(text, plainText)
 
+    def test_RSA_OAEPwithAAD(self):
+        if self.SoftHSMversion < 2:
+            self.skipTest("RSA OAEP only supported by SoftHSM >= 2")
+
+        if self.manufacturer.startswith("SoftHSM"):
+            self.skipTest("SoftHSMv2 returns CKR_ARGUMENTS_BAD. 'AAD' not in expected format or unsupported.")
```

Comment:
I had a look at SoftHSM source code. This feature is not (yet) supported.
In syslog you get the error message: `SoftHSM.cpp(12412): pSourceData must be NULL`

The error message comes from https://github.com/opendnssec/SoftHSMv2/blob/develop/src/lib/SoftHSM.cpp#L12830